src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Header from './navigation/Header';
+import Animals from './component/Animals';
+import MoreAnimalInfo from './component/MoreAnimalInfo';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/more/:moreId" element={<MoreAnimalInfo />} />
+        <Route path="/" element={<Animals />} />
+      </Routes>
```

Comment:
- [ ] Great job so far, one of the requirements is to add a way (component) to filter the presented data on the front page. Kindly add that.

![Screen Shot 2022-09-01 at 12 04 10 PM](https://user-images.githubusercontent.com/59416136/187876346-ea31f8c5-f370-4a8f-b89c-c2bca9820e55.png)

Reply:
@Wes-Isaac  thanks for review i created a list of type of animals